### PR TITLE
Feature/add/region design to multi region feature

### PR DESCRIPTION
**Pull request by havard024**
see #364 for details

### CODE REVIEW

File: OstelcoStyles/OstelcoColor.swift [hunk 1]
```diff
@@ -11,13 +11,15 @@ import SwiftUI
 
 /// Colors from https://app.zeplin.io/project/5c8b989f46989524fb0258ac/styleguide
 public enum OstelcoColor: String, CaseIterable {
+    case azul
```
Comment (sgoodwin) [inline OstelcoStyles/OstelcoColor.swift:14]: Are these the color names florian wanted? I thought we were using use-based names rather than color names?

Comment (havard024) [reply OstelcoStyles/OstelcoColor.swift:14]: There was no name for this color in zeplin, so I asked zeplin to name it for me, can update the name when it comes from design into zeplin

Comment (bloomingbridges) [reply OstelcoStyles/OstelcoColor.swift:14]: Hi guys, 
I'm still in the process of updating the Prod project. So far the only project that uses the new Component Library (and by extension has the most up-to-date styleguide with the right colours) is the OYA MultiSIM UI project.

**Review by sgoodwin — APPROVED** [review]

**Comment by havard024** [comment]
RegionOnboardingCoordinator logic is not working as expected, thus preventing users from completing adding an ESim.

**Pull request merged**